### DISCUSSION TITLE
SHT85 humidity&temperature sensor micropython module

### DISCUSSION THREAD

**Discussion by octaprog7** [micropython/micropython#9150] [Show and tell]
Micropython module for work with SHT85 humidity&temperature sensor from Sensirion. 
https://github.com/octaprog7/sht85

**Comment by scruss** [comment]
Neat! Don't forget to add your modules to [Awesome MicroPython](https://awesome-micropython.com/) so other people can find them

**Comment by mattytrentini** [comment]
I took the liberty of raising [PR68](https://github.com/mcauser/awesome-micropython/pull/68) on Awesome MicroPython 😄

**Reply by mcauser** [reply]
Merged and deployed
